### PR TITLE
Feature/player revive

### PR DESCRIPTION
**Pull request by RyanDefost**

### CODE REVIEW

File: Assets/Scripts/Player/ReSpawning/revivable.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            _healthData.onResurrected.AddListener(OnRevive);
+
+            StartCoroutine(TEMP());
+        }
+
+        IEnumerator TEMP()
+        {
+            yield return new WaitForSeconds(2);
+            _healthData.Kill();
+        }
```
Comment (swzwij) [inline Assets/Scripts/Player/ReSpawning/revivable.cs]: Remove?

File: Assets/Scripts/Player/Pointer.cs [hunk 2]
```diff
@@ -24,8 +24,12 @@ public class Pointer : NetworkBehaviour
         public bool HasClickedUI { get; set; }
         public UnityEngine.Camera Camera => _myCamera;
 
+        public bool CanPoint;
```
Comment (swzwij) [inline Assets/Scripts/Player/Pointer.cs:27]: Does this need to be public?

File: Assets/Scripts/Player/ReSpawning/RespawnTimer.cs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        private int _currentCount;
+
+        private HealthData _healthData;
+
+        private void Start() => _healthData = _player.GetComponent<HealthData>();
+
+        private void OnEnable() => StartCoroutine(CountDown());
+        private void OnDisable() => StopCoroutine(CountDown());
+
+        IEnumerator CountDown()
```
Comment (swzwij) [inline Assets/Scripts/Player/ReSpawning/RespawnTimer.cs]: Missing access modifier

**Review by swzwij — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Player/Pointer.cs (hunk 2 above)
Comment (RyanDefost) [reply Assets/Scripts/Player/Pointer.cs:27]: yeah, it's used in Revivable

**Review by swzwij — APPROVED** [review]

**Pull request merged**